src/modules/fancyzones/tests/UnitTests/JsonHelpers.Tests.cpp

```diff
@@ -520,60 +520,24 @@ namespace FancyZonesUnitTests
     };
 
     TEST_CLASS(ZoneSetDataUnitTest){
-        TEST_METHOD(ToJsonCustom)
```

Comment:
Why these tests were removed?

Reply:
Without zone-count value in json string, this test is basically the same as `ToJsonGeneral` test.. So we would have 2 tests testing same thing.